### DISCUSSION TITLE
04-random.game

### DISCUSSION THREAD

**Discussion by ywanglf** [CastleLab/COMP3021-2021Fall-PA1-Student-Version#35] [Q&A] [answered]
I updated the obf-jar file but then 04-random.game cannot be loaded, though others (01-03.game) still work. May I know why? Below is the error message. Thank you very much.

<img width="645" alt="Screenshot 2021-10-02 at 12 27 58 AM" src="https://user-images.githubusercontent.com/90149606/135655468-209301ad-08f6-4515-a260-3e5e458c7c10.png">
?

**Comment by Derppening** [comment, marked as answer]
That's probably because when I updated the obfuscated JAR with a more conservative implementation of "all-gems reachable", I did not update the maps to ensure that they would still work under the implementation.

This will not affect the grading for your submission.
